clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -119,6 +119,18 @@ class MarkDeviceFunction : public RecursiveASTVisitor<MarkDeviceFunction> {
           SemaRef.addSyclDeviceDecl(Def);
         }
       }
+      if (auto const *FD = dyn_cast<FunctionDecl>(Callee)) {
+        //FIXME: We need check all target specified attributes for error if that
+        //function with attribute can not be called from sycl kernel.  The info
+        //is in ParsedAttr. We don't have to map from Attr to ParsedAttr
+        //currently. Erick is work on that.  Once that in, we need to change
```

Comment:
"Erich is currently working on that in LLVM, once that is committed we need to change this".